server/utils/nunjucksSetup.ts

```diff
@@ -50,4 +50,12 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
   njkEnv.addGlobal('dateFieldValues', function (fieldName: string, errors: ErrorMessages) {
     return dateFieldValues(fieldName, this.ctx, errors)
   })
+
+  njkEnv.addGlobal(
+    'convertObjectsToRadioItems',
+    // eslint-disable-next-line func-names
+    function (items: Array<Record<string, string>>, textKey: string, valueKey: string, fieldName: string) {
```

Comment:
meh.. wondering if avoiding the function name could cause issues with debugging

Reply:
Yeah, this is part of some strangeness around adding nunjucks globals - I can't get access to `this` without using a `function`. If there's a better way, I'd love to know, because I'm not 100% happy putting `eslint-disable`s everywhere

Reply:
Ah, course. I could just do `function sendContextConvertObjectsToRadioItems(....` here